apps/condo/domains/miniapp/components/GlobalApps/GlobalAppsContainer.tsx

```diff
@@ -64,6 +73,60 @@ export const GlobalAppsContainer: React.FC = () => {
         }
     }, [])
 
+    const handleTask = useCallback((message, event) => {
+        const taskRecord = {
+            id: get(message, 'id'),
+            taskId: message.taskId,
+            title: message.taskTitle,
+            progress: message.taskProgress,
+            description: message.taskDescription,
+            status: message.taskStatus,
+            __typename: 'MiniAppTask',
+            sender: event.origin,
+        }
+
+        if (message.taskOperation === 'create') {
+            taskRecord.id = uuidV4()
+            const createMiniAppTask = miniAppTaskUIInterface.storage.useCreateTask({}, () => {
+                addTask({
+                    ...miniAppTaskUIInterface,
+                    record: taskRecord,
+                })
+            })
+            createMiniAppTask(taskRecord)
+
+            for (const iframe of iframeRefs.current) {
+                const targetOrigin = extractOrigin(iframe.src)
+                if (targetOrigin !== taskRecord.sender) continue
+
+                const targetWindow = get(iframe, 'contentWindow', null)
+                if (targetWindow) {
+
+                    sendMessage({
+                        type: TASK_GET_PROCESSING_STATUS,
+                        data: { tasks: [taskRecord] },
+                    }, targetWindow, targetOrigin)
+                }
+            }
```

Comment:
It seems like this code should be executed in `onComplete` handler of `useCreateTask`